src/Polkadot/Extrinsic.cpp

```diff
@@ -35,6 +36,7 @@ static std::map<const std::string, Data> polkadotCallIndices = {
     {stakingNominate, Data{0x07, 0x05}},
     {stakingChill, Data{0x07, 0x06}},
     {utilityBatch, Data{0x1a, 0x02}},
+    {stakingReBond, Data{0x07, 0x13}},
```

Comment:
Indices may be changed if Polkadot or substrate upgrades, this happened on Polymesh.